tests/plugins/autolink/manual/autolink.html

```diff
@@ -24,5 +24,11 @@ <h3>URLs that shouldn't be turned into links</h3>
 </textarea>
 
 <script>
-	CKEDITOR.replace( 'editor1' );
+	CKEDITOR.replace( 'editor1', {
+		on: {
```

Comment:
This can be done easier - you can just put `bender.tools.ignoreUnsupportedEnvironment( 'autolink' );` right after the `<script>` tag. Same for all manuals 🙂

Reply:
I simplified it.